### ISSUE TITLE
Wii U Controller remote play

### ISSUE THREAD

**Issue by athairus** [team-phoenix/Phoenix#105]
Possible thanks to this library:

http://libdrc.org/index.html

**Comment by RonnChyran** [comment]
libdrc only works on Linux, if somehow you can port it to Windows that
would be amazing

**Comment by athairus** [comment]
Yeah, that's definitely going to be a requirement if Phoenix were to use the library. All 3 platforms are first-class citizens for this project.

**Comment by athairus** [comment]
The "driver" portion of this project is a small change to `net/mac80211/iface.c` that exposes a network time sync object internal to the module as a file that can be accessed from user mode. I don't yet know if this is already done in Windows or OS X

I've gotten this working on an Arch VM. Here's what needs to be done to close this issue:
- Linux: DKMS for the patched module so it'll survive kernel updates
- Windows: Check out their work on `wudrnet`, see how far along that is (and also buy the necessary hardware)
- Make available custom `hostapd` and [`netboot`](https://github.com/ITikhonov/netboot) as an add-on download for Phoenix 
- Write necessary UI to facilitate and inform the user about the connection to the Wii U gamepad
- Determine for all 3 platforms what cards are supported
- Make it clear to the user that they'll probably be unable to use their WiFi to connect to the Internet while using it to connect to the gamepad
- Make necessary backend changes `DRCOutput`

**Comment by athairus** [comment]
Alternate method: Just stream over HTTP both ways. Implemented with UsendMii and Gamepad to PC:

https://gbatemp.net/threads/usendmii.342772/
https://gbatemp.net/threads/gamepad-to-pc-controller-release-tutorial.342360/

I doubt the latency is great, but maybe it'll make for a good fallback method for those who don't want to buy the necessary hardware?

**Comment by WolfLink115** [comment]
Just wondering, but has anyone gotten libdrc to work yet, if so, do you need a WiiU for it to work??

**Comment by thefloppydriver** [comment]
@WolfLink115 I'm patching and setting up instructions on how to use the gamepad as a second screen and controller using libdrc, check my github for updates, I'll be posting my forks of the project and my new project there soon. https://youtu.be/_whlwABk7HE

**Comment by WolfLink115** [comment]
@thefloppydriver Hey, thanks for responding. Sadly I do not have my Wii U gamepad anymore. However when I get another one if at all, I might try libdrc again.

Thanks again!
